packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsScreen.tsx

```diff
@@ -40,7 +46,7 @@ const CompareAccountsScreen = () => {
                         />
                     ))}
                     {/* Renders cTrader data */}
-                    {mt5Accounts?.length && hasCTraderAccountAvailable && ctraderAccount && (
+                    {!isEU && mt5Accounts?.length && hasCTraderAccountAvailable && ctraderAccount && (
```

Comment:
can you refactor this and make a component for cTrader outside and just render  it here?

```suggestion
                    Const CtraderCompareAccount  = (is_eu) => {
                         if (isEU) return null;
                         ...
                    }
```